lib/claide/command/plugin_manager.rb

```diff
@@ -54,7 +54,9 @@ def self.specifications
       #
       def self.specification(path)
         matches = Dir.glob("#{path}/*.gemspec")
-        spec = Gem::Specification.load(matches.first) if matches.count == 1
+        spec = silence_streams(STDERR) do
+          Gem::Specification.load(matches.first)
```

Comment:
Yes map and collect are the same. No it doesn't seem that rubygem handle
correctly the way Claide load spec.

Cheers,

Clem

On Thursday, January 22, 2015, Samuel E. Giddins <notifications@github.com>
wrote:

> In lib/claide/command/plugin_manager.rb
> <https://github.com/CocoaPods/CLAide/pull/44#discussion_r23388652>:
>
> > @@ -54,7 +54,9 @@ def self.specifications
> >        #
> >        def self.specification(path)
> >          matches = Dir.glob("#{path}/*.gemspec")
> > -        spec = Gem::Specification.load(matches.first) if matches.count == 1
> > +        spec = silence_streams(STDERR) do
> > +          Gem::Specification.load(matches.first)
>
> Does rubygems not have any way to do this already?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CocoaPods/CLAide/pull/44/files#r23388652>.
>